qa-tests-backend/src/main/groovy/services/ImageService.groovy

```diff
@@ -37,11 +37,14 @@ class ImageService extends BaseService {
 
     static scanImage(String image, Boolean includeSnoozed = true, Boolean force = false) {
         try {
-            return getImageClient().scanImage(ImageServiceOuterClass.ScanImageRequest.newBuilder()
-                    .setImageName(image)
-                    .setIncludeSnoozed(includeSnoozed)
-                    .setForce(force)
-                    .build())
+            def req = ImageServiceOuterClass.ScanImageRequest.newBuilder()
+                .setImageName(image)
+                .setIncludeSnoozed(includeSnoozed)
+                .setForce(force)
+                .build()
+            return withRetry(1, 15) {
+                return getImageClient().scanImage(req)
+            }
```

Comment:
Based on the failed test logs, looks like this is still invalid, perhaps this will do the trick:

```suggestion
def req = ImageServiceOuterClass.ScanImageRequest.newBuilder()
                .setImageName(image)
                .setIncludeSnoozed(includeSnoozed)
                .setForce(force)
                .build()

            ImageOuterClass.Image imageScan = null
            withRetry(1, 15) {
                imageScan = getImageClient().scanImage(req)
            }
            return imageScan
```
withRetry has a void return, so noting was being output (saw a similar use of retry in `ImageScanningTest.groovy`)